pkg/parent/mount/mount_linux.go

```diff
@@ -0,0 +1,82 @@
+// +build linux
+
+package mount
+
+import (
+	"bufio"
+	"fmt"
+	"io"
+	"os"
+	"strings"
+)
+
+const (
+	/* 36 35 98:0 /mnt1 /mnt2 rw,noatime master:1 - ext3 /dev/root rw,errors=continue
+	   (1)(2)(3)   (4)   (5)      (6)      (7)   (8) (9)   (10)         (11)
+
+	   (1) mount ID:  unique identifier of the mount (may be reused after umount)
+	   (2) parent ID:  ID of parent (or of self for the top of the mount tree)
+	   (3) major:minor:  value of st_dev for files on filesystem
+	   (4) root:  root of the mount within the filesystem
+	   (5) mount point:  mount point relative to the process's root
+	   (6) mount options:  per mount options
+	   (7) optional fields:  zero or more fields of the form "tag[:value]"
+	   (8) separator:  marks the end of the optional fields
+	   (9) filesystem type:  name of filesystem of the form "type[.subtype]"
+	   (10) mount source:  filesystem specific information or "none"
+	   (11) super options:  per super block options*/
+	mountinfoFormat = "%d %d %d:%d %s %s %s %s"
+)
+
+// Parse /proc/self/mountinfo because comparing Dev and ino does not work from
+// bind mounts
+func parseMountTable() ([]*Info, error) {
```

Comment:
Was this code copied from the moby "mount" pkg?

Should we make that package easier to import, you think? (separate repo or separate module if we're moving to go modules?)

Reply:
It is from runc: https://github.com/rootless-containers/rootlesskit/pull/122/files#diff-eb54b121747b544dfaf8954b8474c615R1

I think this amount of copypasta is fine. Separating the repo is rather likely to cause vendor hell.

Reply:
And, looks like runc copied it from moby 😓, so now it's in three locations https://github.com/moby/moby/blob/c4cf72bad325f90ebf021d7cb52c262992a687c7/pkg/mount/mountinfo_linux.go#L120-L130